### PR TITLE
Add information about Canarytokens and the Canarytokens Factory

### PR DESCRIPTION
**Pull request by jayjb**

### CODE REVIEW

File: docs/canarytokens/factory.md [hunk 1]
```diff
@@ -57,12 +68,67 @@ endpoints:
...
+      - name: trigger_name
+        required: false
+        type: string
+        description: Name of the trigger we will create (required when creating sql tokens with `trigger_type` as `INSERT`, `UPDATE` or `DELETE`)
+      - name: table_name
+        required: false
+        type: string
+        description: Name of table that will activate the trigger when the `trigger_type` action is executed on this table (required when creating
+                     sql tokens with `trigger_type` as `INSERT`, `UPDATE` or `DELETE`)
+        expected_values: zxc, asd, qwe
```
Comment (nickrohrbs) [inline docs/canarytokens/factory.md:114]: I'm presuming there are no expected values here (especially `zxc` being one of them)?

Comment (jayjb) [reply docs/canarytokens/factory.md:114]: Ah let me remove that

File: docs/canarytokens/actions.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 ---
 endpoints:
+  list_canarytokens:
+    name: List available Canarytokens
+    url: /api/v1/canarytokens/list
+    method: GET
+    description: List the available Canarytokens on your Canary Console.
```
Comment (nickrohrbs) [inline docs/canarytokens/actions.md:7]: Let's use the present tense of the verb: `Lists the available..`

File: docs/canarytokens/actions.md [hunk 3]
```diff
@@ -225,6 +270,91 @@ These are a collection of endpoints that allow you mint new, interact with, and
 
 </APIEndpoints>
 
+## List Canarytokens
+
+::: tip
+The values returned by this Canarytokens API correspond to the `kind` parameter used to create
+Canarytokens. As an example, if I wanted to create a Cloned Web Canarytoken, I would check the
```
Comment (nickrohrbs) [inline docs/canarytokens/actions.md:277]: Let's default to the second voice as a convention (you/your vs I).

> As an example, if you wanted to create a Cloned Web Canarytoken, you would check the response from this endpoint and use `cloned-web` as the `kind`.

File: docs/canarytokens/factory.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 ---
 endpoints:
+  list_factory_canarytokens:
+    name: List Canarytokens available via Canarytoken Factory
+    url: /api/v1/canarytokens/factory/list
+    method: GET
+    description: List the Canarytokens available via your Canarytokens Factory.
```
Comment (nickrohrbs) [inline docs/canarytokens/factory.md:7]: Present tense: `Lists the..`

File: docs/canarytokens/factory.md [hunk 5]
```diff
@@ -80,11 +145,87 @@ The Canarytoken factory gives you a limited use key that is able to create other
 
 </APIEndpoints>
 
+## List Canarytokens available via Canarytoken Factory
+
+::: tip
+The values returned by this Canarytokens Factory API correspond to the `kind` parameter used to create
+Canarytokens via the Canarytokens Factory. As an example, if I wanted to create a Cloned Web Canarytoken, I would check the
```
Comment (nickrohrbs) [inline docs/canarytokens/factory.md:152]: Ditto as above using second voice.

**Review by nickrohrbs — CHANGES_REQUESTED** [review]
We're making some changes to our styling guide for the docs, including using second voice and present tense in descriptions.

**Pull request merged**